### PR TITLE
225 kilogram as price

### LINKED ISSUE #225: Kilograms recognized as prices
In bill 25KA7rWWmhStXDEsb.pdf kilograms are recognized as prices:

screen shot 2017-04-26 at 18 15 28

28,80 kg is recognized as price 28,80.

### PR DESCRIPTION
**Pull request by Tr4in**
I'm not satisfied with that, because it only works if numbers are followed by kg!

I tried PriceTerm.right_before(<kg-word>) and it detects:
28,80
3,76
4,24
366,00

The first three aren't prices but the last is one! AND 36,80 was not detected!

selection_015

Closes #225

### CODE REVIEW

File: lib/models/price_term.rb [hunk 1]
```diff
@@ -26,7 +26,16 @@ def add_word(word)
...
-    @term_builder.valid?
+    if @term_builder.valid?
+      # Find the word which is right after this PriceTerm
+      word_right_after = Word.right_after(self)
+
+      # Get the text only if the word right after exists
+      text = word_right_after.text unless word_right_after.nil?
+
+      # Check if the text is kg
+      text != 'kg'
```
Comment (clemenshelm) [inline lib/models/price_term.rb]: Könntest du kürzen auf `word.right_after&.text != 'kg'`.

http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/

File: lib/detectors/price_detector.rb [hunk 2]
```diff
@@ -27,35 +27,33 @@ def self.filter
...
         end
       end
+
+      def initialize_new_term(regex, after_each_word, max_words)
+        PriceTerm.new(
+          regex: regex,
+          after_each_word: after_each_word,
+          max_words: max_words
+        )
+      end
```
Comment (clemenshelm) [inline lib/detectors/price_detector.rb]: Sollte überflüssig werden, wenn du den neuen Master reinmergest 😄

**Review by clemenshelm — CHANGES_REQUESTED** [review]

File: lib/models/word.rb [hunk 3]
```diff
@@ -12,6 +12,10 @@ def next
     Word[id + 1]
   end
 
+  def previous
+    Word[id - 1]
+  end
```
Comment (clemenshelm) [inline lib/models/word.rb]: Bitte entfernen.

**Review by clemenshelm — APPROVED** [review]

**Review by tamacodechi — APPROVED** [review]
This is now mergeable. Any more feedback @clemenshelm ?

**Comment by clemenshelm** [comment]
Meeeerged! 😄

**Pull request merged**